documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-application-credential-authentication_{context}"]
+= Using application credential authentication with an {osp} source provider
+
+You can use application credential authenication instead of username/password authentication when you use {product-first} to migrate a virtual machine that has an {osp} source provider.
+
+{project-short} supports both of the following types of token authentication:
+
+* Application credential ID
+* Application credential name
+
+For each type of application credential authentication, you need to use data from OpenStack to create a `Secret` manifes
+
+.Prerequisites
+
+Have an {osp} account.
+
+.Procedure
+
+. In the dashbord of the {osp} web console, click your name and then cick *Openstack RC file* to download the `<openstack_password_authentication>` file.
+. To get the data needed for token authetication run the following command:
++
```

Comment:
It looks like the path is different for the downstream Red Hat OpenStack: Expand the `Project` dropdown menu and click `API Access`, then expand the `Download Openstack RC file` and click `Openstack RC file` (there's also de possibility to download the clouds.yaml file).